metamod/Makefile

```diff
@@ -179,7 +179,7 @@ CCO += -flto -fvisibility=hidden
 ifeq "$(TARGETTYPE)" "amd64"
 	CCOPT_ARCH =
 else
-	CCOPT_ARCH = -march=i686 $(MCPU)=generic -msse -msse2 -msse3
+	CCOPT_ARCH = -march=i686 $(MCPU)=generic -msse2 -msse3
```

Comment:
I checked how this selection works in GCC source code and look like even the -msse2 is not needed as -msse3 implicitly selects both SSE and SSE2 features in addition to SSE3:  
 https://github.com/gcc-mirror/gcc/blob/269295911ad9491222359fae0291253b935e1c36/gcc/common/config/i386/i386-common.c#L44